.github/workflows/cmake.yml

```diff
@@ -187,6 +187,24 @@ jobs:
       adapter_name: NATIVE_CPU
       runner_name: NATIVE_CPU
 
+  # Native CPU jobs are here to force the loader to be used
+  combined-opencl-native-cpu:
+    name: OpenCL + Native CPU
+    uses: ./.github/workflows/build-hw-reusable.yml
+    with:
+      adapter_name: OPENCL
+      other_adapter_name: NATIVE_CPU
+      runner_name: OPENCL
+      platform: "Intel(R) OpenCL"
```

Comment:
[NIT] I feel it would be more clear to name the job a name that would relate that we are testing the loader here and explain in a comment that we are enabling multiple adapters to enable the loader to be used.